openslides_backend/action/action.py

```diff
@@ -42,6 +42,7 @@
 from .relations.typing import FieldUpdateElement, ListUpdateElement
 from .util.assert_belongs_to_meeting import assert_belongs_to_meeting
 from .util.typing import ActionData, ActionResultElement, ActionResults
+from ..services.datastore.with_database_context import with_database_context
```

Comment:
Imported but not used?

Reply:
Yes, at the beginning i had though about creating the context here, probably a leftover. It's still WIP after all :D